Wikipedia/Code/ColumnarCollectionViewController.swift

```diff
@@ -39,11 +39,15 @@ class ColumnarCollectionViewController: UICollectionViewController, Themeable {
                 cellWithSubItems.deselectSelectedSubItems(animated: animated)
             }
         }
+        NotificationCenter.default.addObserver(forName: .UIContentSizeCategoryDidChange, object: nil, queue: nil) {_ in
+            self.collectionView?.reloadData()
```

Comment:
this can probably be put inside of `traitCollectionDidChange`

Reply:
ah thanks lemme try that...

Reply:
Fixed!